### ISSUE TITLE
Internal error: subprocess + os.environ

### ISSUE THREAD

**Issue by starsep** [microsoft/pyright#535]
Following program causes pyright internal error

**Example**
``` python
import os
import subprocess

subprocess.check_call([], env=os.environ)
```

**Pyright output**
```
typingsPath PATH/typings is not a valid directory.
Searching for source files
Found 1 source file
An internal error occurred while while performing type checking for PATH/example.py: Error: Debug Failure. False expression.
    at TypeVarMap.set (webpack:///./src/analyzer/typeVarMap.ts?:38:17)
    at assignTypeToTypeVar (webpack:///./src/analyzer/typeEvaluator.ts?:7591:28)
    at canAssignType (webpack:///./src/analyzer/typeEvaluator.ts?:7685:22)
    at verifyTypeArgumentsAssignable (webpack:///./src/analyzer/typeEvaluator.ts?:7554:30)
    at canAssignClassWithTypeArgs (webpack:///./src/analyzer/typeEvaluator.ts?:7512:18)
    at canAssignClass (webpack:///./src/analyzer/typeEvaluator.ts?:7422:24)
    at canAssignType (webpack:///./src/analyzer/typeEvaluator.ts?:7828:22)
    at eval (webpack:///./src/analyzer/typeEvaluator.ts?:7756:21)
    at Array.forEach (<anonymous>)
    at canAssignType (webpack:///./src/analyzer/typeEvaluator.ts?:7755:31)
PATH/example.py
```

**VS Code extension or command-line**
I use command-line, `pyright example.py`
> $ pyright --version
> pyright 1.1.24

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is fixed in 1.1.25, which I just published.
